scripts/test/shared.py

```diff
@@ -524,3 +524,13 @@ def with_pass_debug(check):
         else:
             if 'BINARYEN_PASS_DEBUG' in os.environ:
                 del os.environ['BINARYEN_PASS_DEBUG']
+
+
+# checks if we are on windows, and if so logs out that a test is being skipped,
+# and returns True. This is a central location for all test skipping on
+# windows, so that we can easily find which tests are skipped.
+def skip_if_on_windows(name):
+    if get_platform() == 'windows':
+        print('skipping test "%s" on windows' % name)
+        return True
+    return False
```

Comment:
The idea is that this would be called when we know a test currently fails on windows. It just returns true/false so it's up to the calling code to actually skip the test.

I plan to use it in this PR to disable all failing windows tests, which should let us enable the majority of tests at least (I hope).

Reply:
What I meant was, this returns `True` for everything on Windows now even without checking names.. So do you mean this is a placeholder for now and you are gonna add real file names later?

Reply:
No, this is the actual function. It's only called when we should definitely skip what we are currently doing if we are on windows. So the caller decides which test or part of a test to skip, and this just logs and returns True, and provides a simple way to search for all windows skips.

See e.g. the wasm2js test change https://github.com/WebAssembly/binaryen/pull/3035/files#diff-611fb24eb3395253b8c42c20f7238e8e

Reply:
Oh, I see. Thanks. I was confused.